File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/catalog/setdetail/CatalogSetDetailClickListener.java
package com.lucagiorgetti.surprix.ui.mainfragments.catalog.setdetail;

import android.widget.ImageView;
import com.lucagiorgetti.surprix.model.Surprise;

public interface CatalogSetDetailClickListener extends SetDetailClickListener{
    void onSurpriseAddedToDoubles(Surprise s);

    void onSurpriseAddedToMissings(Surprise s);
}

File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/catalog/setdetail/CatalogSetDetailClickListener.kt
package com.lucagiorgetti.surprix.ui.mainfragments.catalog.setdetail

import com.lucagiorgetti.surprix.model.Surprise

interface CatalogSetDetailClickListener : SetDetailClickListener {
    fun onSurpriseAddedToDoubles(s: Surprise)
    fun onSurpriseAddedToMissings(s: Surprise)
}